src/coreclr/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.Core.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*=============================================================================
+**
+**
+**
+** Purpose: Class for creating and managing a threadpool
+**
+**
+=============================================================================*/
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+namespace System.Threading
+{
+    internal sealed partial class CompleteWaitThreadPoolWorkItem : IThreadPoolWorkItem
```

Comment:
`class CompleteWaitThreadPoolWorkItem` should be moved under libraries
CompleteWaitThreadPoolWorkItem.CompleteWait should be deleted (dead code)